models/org.go

```diff
@@ -494,7 +494,11 @@ func removeOrgUser(sess *xorm.Session, orgID, userID int64) error {
 	if err != nil {
 		return fmt.Errorf("AccessibleReposEnv: %v", err)
 	}
-	repoIDs, err := env.RepoIDs(1, org.NumRepos)
+	count, err := org.GetRepositoryCount()
```

Comment:
Count is not needed here, just make env.RepoIDs accept -1 as no limit condition or pass large enough limit

Reply:
Done,